src/core/symbology-ng/qgssymbollayerutils.cpp

```diff
@@ -3986,6 +4003,14 @@ double QgsSymbolLayerUtils::rescaleUom( double size, QgsUnitTypes::RenderUnit un
           scale = 1 / 0.28;
           roundToUnit = true;
           break;
+        case QgsUnitTypes::RenderInches:
+          scale = 1 / 0.28 * 25.4;
+          roundToUnit = true;
+          break;
+        case QgsUnitTypes::RenderPoints:
+          scale = 90. /* dots per inch according to OGC SLD */ / 72. /* points per inch */ ;
+          roundToUnit = true;
+          break;
         // we don't have a good case for map units, as pixel values won't change based on zoom
         default:
```

Comment:
Could we remove the "default" here now and explicitly list the units that can't be mapped? (then a warning will be thrown if more units are added in future)